src/Daemon/BaseDaemon.cpp

```diff
@@ -355,23 +355,23 @@ class SignalListener : public Poco::Runnable
         String calculated_binary_hash = getHashOfLoadedBinaryHex();
         if (daemon.stored_binary_hash.empty())
         {
-            LOG_FATAL(log, "Calculated checksum of the binary: {}."
-                " There is no information about the reference checksum.", calculated_binary_hash);
+            LOG_FATAL(log, "Integrity check of the executable skipped because the reference checksum could not be read."
```

Comment:
Just a question if it makes sense to reuse this in programs/server/Server.cpp

Reply:
Yeah, there is some copypasta. Reuse is probably not worth the trouble as its in two files but what I can do is to add a comment to keep the text at least in sync.